phpstan.neon

```diff
@@ -4,6 +4,12 @@ includes:
 parameters:
 	# Level 9 is the highest level
 	level: max
+	tips:
+	  treatPhpDocTypesAsCertain: false
 
 	paths:
 		- src/
+
+	ignoreErrors:
+	  -
+	      identifier: function.alreadyNarrowedType
```

Comment:
The way the code of this project was written, it assumed (or it was wrote on purpose) PHPDoc types were not certain. So we get a conflict here since PHPStan by default treats it as `true`.